rps.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
@rrykowski this is great - now what we want to do is port this content over to our view templates that we intend to be responsible for showing you the game. There will be two parts to this, because some of this is stuff we want to define in what rails calls a `layout` - these will always wrap your view templates with some standard content that should be shared across views (e.g. dependencies like bootstrap, a navigation UI, standard header, etc). Then the content that is specific to the game will go in our `games/rps/new.html.erb` template since that's the page where we want people to actually be able to play the game.